dotnet/targets/WorkloadManifest.tvOS.template.json

```diff
@@ -0,0 +1,27 @@
+{
+	"version": "@VERSION@",
+	"workloads": {
+		"microsoft-@PLATFORM_LOWERCASE@-sdk-full": {
+			"description": "Microsoft @PLATFORM@ SDK",
```

Comment:
If you add extends `microsoft-net-runtime-tvos`, your workload will pull along the tvOS runtime packs and aot compilers as part of the install